features/editor/components/editor-header.tsx

```
@@ -34,55 +50,141 @@ export const EditorSaveButton = ({ workflowId }: { workflowId: string }) => {
+  const isGuest = workflowId.startsWith("guest-");
+  const { data: workflow } = useQuery(
+    isGuest
+      ? {
+          queryKey: ["guest-workflow", workflowId],
+          queryFn: () => getGuestWorkflow(workflowId),
+        }
+      : trpc.workflows.getOne.queryOptions({ id: workflowId })
+  );
+  const { data: authData } = useQuery({
+    queryKey: ["session"],
+    queryFn: () => authClient.useSession(),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)